sigopt_sklearn/search.py

```diff
@@ -227,7 +228,19 @@ def _check_bounds():
         # generate sigopt experiment parameters
         return [_transform_param(name, bounds) for (name, bounds) in param_domains.items()]
 
-    def _create_sigopt_exp(self, conn, folds):
+    def create_experiment(self, num_folds):
```

Comment:
so, if create_experiment is a method on an already constructed SigOptSearchCV, then it can't be used to "prepopulate" the experiment. can we make this static?